nixos/modules/programs/foliate.nix

```diff
@@ -0,0 +1,117 @@
+{ lib, pkgs, config, ... }:
+
+with lib; {
+  options.programs.foliate = {
+    enable = mkEnableOption "Foliate - A simple and modern eBook reader";
+
+    footer = {
```

Comment:
Those are basically only opinionated theming settings. It would be better to use freeform settings or just let people write into a Multiline string.

Reply:
I find value in using enumerations because they are discoverable and the possible values are known to be correct. Using freeform settings makes it easy to introduce wrong values (which can break the application), and using multiline strings makes it easy to introduce a syntax error or wrong values (which will avoid the settings or the application from working)

as you mentioned in the other comment, dconf is a special case. To me managing the options as enumerations seems safer. What we could do is having a `programs.foliate.extraConfig` option so that users can inject the values that are not exposed in this interface. What do you think?

Reply:
> using multiline strings makes it easy to introduce a syntax error

Many modules have extraConfig and if you paste invalid config into there, the programs will break but with that we only need one option **not** 10 individually ones